examples/docker/solara/app.py

```diff
@@ -1,31 +1,93 @@
+from typing import Optional, cast
+
+import pandas as pd
+import requests
+
 import solara
+import solara.express as solara_px  # similar to plotly express, but comes with cross filters
+import solara.lab
+from solara.components.columns import Columns
+from solara.components.file_drop import FileDrop
+
+import spacy
+import pytextrank
+
+
+try:
+    # fails on pyodide
```

Comment:
we won't run this using pyodide, I think we can remove it

Reply:
[fixed](https://github.com/ploomber/doc/pull/26/files#diff-f93f544af0cb5c95146bd5be948e6e37d6cff76c1e8e7b56699b54c2db3a2a92R16-R18)

Reply:
do we still need the try, catch?

Reply:
Forgot to remove. 
[Fixed](https://github.com/ploomber/doc/pull/26/files#diff-f93f544af0cb5c95146bd5be948e6e37d6cff76c1e8e7b56699b54c2db3a2a92R15-R16)